VideoLocker/res/layout/fragment_course_unit_video.xml

```diff
@@ -28,47 +28,46 @@
     </LinearLayout>
 
     <LinearLayout
+        android:id="@+id/player_container"
         android:layout_width="match_parent"
         android:layout_height="200dp"
-        android:orientation="vertical"
-        android:id="@+id/player_container">
+        android:orientation="vertical">
 
     </LinearLayout>
 
     <LinearLayout
```

Comment:
display differently = yes
lemme change the layout to centrally align the icon+text.